### ISSUE TITLE
centralising API calls

### ISSUE THREAD

**Issue by phillimorland** [liqd/a4-meinberlin#4446]
discussion was started but conclusion not recorded: https://github.com/liqd/a4-meinberlin/pull/4114
@khamui I can't remember the conclusion but we would probably be better with a vanilla api.js like in a4 then a .jsx that can only be used for react right?have closed it for now, maybe we can discuss further while we do the mB sprints

**Comment by khamui** [comment]
@phillimorland yes i think you are right. We were even thinking to simplify it and use just one api.js in a4? Bringing @goapunk  @fuzzylogic2000 @Rineee into this discussion. How should we continue with this?
